.editorconfig

```diff
@@ -10,3 +10,6 @@ insert_final_newline = true
 indent_size = 2
 indent_style = space
 trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Why don't we trim whitespaces for `.md` files?

Reply:
In a paragraph two blank spaces at end of the line could be used to force new line -> https://commonmark.org/help/tutorial/03-paragraphs.html

Reply:
TIL :+1:

Reply:
In other cases, I believe Prettier will trim unnecessary space, so style wise it's safe